tests/drivers/spi/spi_slave/src/main.c

```diff
@@ -0,0 +1,454 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/ztest.h>
+
+#define SPI_MODE (SPI_MODE_CPOL | SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE)
+#define SPIM_OP	 (SPI_OP_MODE_MASTER | SPI_MODE)
+#define SPIS_OP	 (SPI_OP_MODE_SLAVE | SPI_MODE)
+
+static struct spi_dt_spec spim = SPI_DT_SPEC_GET(DT_NODELABEL(dut_spi_dt), SPIM_OP, 0);
+static struct spi_dt_spec spis = SPI_DT_SPEC_GET(DT_NODELABEL(dut_spis_dt), SPIS_OP, 0);
+
+static struct k_poll_signal async_sig = K_POLL_SIGNAL_INITIALIZER(async_sig);
+static struct k_poll_event async_evt =
+	K_POLL_EVENT_INITIALIZER(K_POLL_TYPE_SIGNAL, K_POLL_MODE_NOTIFY_ONLY, &async_sig);
+
+#define MEMORY_SECTION(node)                                                                       \
+	COND_CODE_1(DT_NODE_HAS_PROP(node, memory_regions),                                        \
+		    (__attribute__((__section__(                                                   \
+			    LINKER_DT_NODE_REGION_NAME(DT_PHANDLE(node, memory_regions)))))),      \
+		    ())
+
+static uint8_t spim_buffer[32] MEMORY_SECTION(DT_NODELABEL(dut_spi));
```

Comment:
You could use `DT_BUS(DT_NODELABEL(dut_spi_dt))` here to avoid requiring another node label.

Reply:
done, removed `dut_spi` label.